src/appsignal/client.py

```diff
@@ -48,6 +51,19 @@ def start(self) -> None:
         else:
             logger.info("AppSignal not starting: no active config found")
 
+    def stop(self) -> None:
+        logger.info("Stopping AppSignal")
+        working_dir = self._config.option("working_dir") or "/tmp/appsignal"
```

Comment:
It seems the config option's name is [`working_directory_path`](https://github.com/appsignal/appsignal-python/blob/10422663a4723e424f7c5a663269332848f18e69/src/appsignal/config.py#L51).